content/topics/file-and-directory-naming/python/_index.md

```diff
@@ -60,7 +60,7 @@ Is any of the examples below up to our recommended naming standard?
  > my_file .py
```

Comment:
just to double check is this meant to have a space in it? (i.e to show that it's incorrect) or is it meant to be the one ok example on the list?

Reply:
Its meant to be like that.